app.json

```diff
@@ -142,6 +142,14 @@
     "GITHUB_ACCESS_TOKEN": {
       "description": "An API token for GitHub, optional for self hosted (optional)",
       "required": false
+    },
+    "TEAM_NAME": {
```

Comment:
I think this might be confusing because the team name is determined from the login authentication, even on the hosted version. Better to remove as it's only used for ALT text anyway